Orange/widgets/visualize/owsieve.py

```diff
@@ -192,6 +187,7 @@ def set_data(self, data):
         self.vizrank_button.setEnabled(
             self.data is not None and len(self.data) > 1 and
             len(self.data.domain.attributes) > 1)
+        self.vizrank_button.setEnabled(self.data is not None and not self.data.is_sparse())
```

Comment:
This defeats checking for `len(self.data) > 1` and `len(self.data.domain.attributes) > 1` in the previous line since we just check for sparsity.

Please add `and not self.data.is_sparse()` to the condition in the line above.